enclave-tls/dist/rpm/enclave_tls.spec

```diff
@@ -34,6 +34,17 @@ enclave-tls is a protocol to establish secure and trusted channel by integrating
 %setup -q -n %{PROJECT}-%{version}
 
 %build
+# we cann't install SGX SDK through 'yum install' in centos, so that wo check the go version in the '%build' section rather than in the 'BuildRequires' section.
```

Comment:
Please change to `If the SGX SDK is not prepared well in build environment, stop the build.`